### PR TITLE
Fetch diagram api in front

### PR DESCRIPTION
**Pull request by Nicklas-Holmqvist**

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { ThemeProvider } from '@material-ui/core/styles';
 import { LogsProvider } from './context/logs';
 import { UsersProvider } from './context/users';
+import { DiagramProvider } from './context/diagram';
 import { Route, BrowserRouter, Routes } from 'react-router-dom';
```
Comment (olivernygren) [inline client/src/App.tsx:5]: separera imports

File: client/src/components/Diagram/Diagram.tsx [hunk 2]
```diff
@@ -1,43 +1,126 @@
...
+    CategoryScale,
+    LinearScale,
+    PointElement,
+    LineElement,
+    Title,
+    Tooltip,
+    Legend,
+  } from 'chart.js';
+import { Line } from 'react-chartjs-2';
+import { Button, Grid, Typography } from '@material-ui/core';
```
Comment (olivernygren) [inline client/src/components/Diagram/Diagram.tsx:21]: separera imports

File: client/src/App.tsx (hunk 1 above)
Comment (Nicklas-Holmqvist) [reply client/src/App.tsx:5]: Done

File: client/src/components/Diagram/Diagram.tsx (hunk 2 above)
Comment (Nicklas-Holmqvist) [reply client/src/components/Diagram/Diagram.tsx:21]: done

**Pull request merged**